.ci/merge-fixes.sh

```diff
@@ -2,7 +2,9 @@
 # Merge open PRs from sagemath/sage labeled "blocker".
 REPO="sagemath/sage"
 GH="gh -R $REPO"
-PRs="$($GH pr list --label "p: blocker / 1" --json number --jq '.[].number')"
+mkdir -p upstream
+PRs="$($GH pr list --label "p: blocker / 1" --json number --jq '.[].number' | tee upstream/ci-fixes.txt)"
```

Comment:
Why not add `--label "s: positive review"`? Multiple `--label` arguments work just fine.

Reply:
We can do that later when the label-warring phase has been outgrown